authorize/run.go

```diff
@@ -0,0 +1,152 @@
+package authorize
+
+import (
+	"context"
+	"time"
+
+	"google.golang.org/protobuf/types/known/emptypb"
+	"gopkg.in/tomb.v2"
+
+	"github.com/cenkalti/backoff"
+
+	"github.com/pomerium/pomerium/internal/grpc/databroker"
+	"github.com/pomerium/pomerium/internal/log"
+)
+
+// Run runs the authorize server.
+func (a *Authorize) Run(ctx context.Context) error {
+	t, ctx := tomb.WithContext(ctx)
```

Comment:
Is there any reason to use `tomb` here instead of `errgroup`, I see we're using `errgroup` for this kind of task in other places.

Reply:
yes there was a reason but I don't remember what it was. I will switch to errgroup.